client/src/context/primitives/boardContextHelper.jsx

```diff
@@ -0,0 +1,71 @@
+import { useRef } from 'react'
+import {
+    Button, TextField, Dialog, DialogContent,
+    DialogTitle, Box
+} from '@material-ui/core';
+import { FetchBoard } from '../../helpers/APICalls/board';
+
+
+export const resolveBoard = (boardColumns, columnItems) => {
+    return boardColumns.sort((colA, colB) => colA.index - colB.index).reduce(
+        (previous, columnId) => ({
+            ...previous,
+            [columnId.name]: [...columnItems.filter((item) => item.columnId === columnId)],
+        }), {});
+}
+
+export const processBoard = async (boards, id) => {
+    const board = id ? boards.find(x => x.id == id) : boards[0];
+    const boardColumns = board.columns;
+    const columnItems = [];
+    boardColumns.forEach((column) => columnItems.push(...column.tasks));
+    columnItems.forEach((column_1) => column_1.columnId = boardColumns.find((col) => col.index == column_1.columnId));
```

Comment:
You can use `column` instead of `column_1` here if you were concerned about name conflicts - either way let's use a descriptive name in *`camelCase`* for it 🙌🏼